### ISSUE TITLE
last one bug over ZZ for today

### ISSUE THREAD

**Issue by jakobkroeker** [Macaulay2/M2#244]
```
rng = ZZ[x_1, x_2, x_3 ]
I   = ideal(6*x_3,2*x_3-12,-3*x_1+1)
gI  = ideal gens gb  I;
ggI = ideal gens gb gI;
gI
ggI 
assert(numColumns (gens gI) == numColumns(gens ggI)); --fails
```
output
```
i22 : rng = ZZ[x_1, x_2, x_3 ]

o22 = rng

o22 : PolynomialRing

i23 : I   = ideal(6*x_3,2*x_3-12,-3*x_1+1)

o23 = ideal (6x , 2x  - 12, - 3x  + 1)
               3    3           1

o23 : Ideal of rng

i24 : gI  = ideal gens gb  I;

o24 : Ideal of rng

i25 : ggI = ideal gens gb gI;

o25 : Ideal of rng

i26 : gI

o26 = ideal (12, 2x , 3x  - 1, x x  + x )
                   3    1       1 3    3

o26 : Ideal of rng

i27 : ggI 

o27 = ideal (4, 2x , x  + 1)
                  3   1

o27 : Ideal of rng

i28 : assert(numColumns (gens gI) == numColumns(gens ggI)); --fails
stdio:158:1:(3): error: assertion failed

```

**Comment by jakobkroeker** [comment]
two simpler(?) failing examples:
```
rng = ZZ[x_1, x_2, x_3 ]
I   = ideal(6*x_3,x_3-12,-3*x_1+1)
gI  = ideal gens gb  I;
ggI = ideal gens gb gI;
gI
ggI 


rng = ZZ[x_1, x_2, x_3 ]
I   = ideal(2*x_3,x_3-12,-3*x_1+1)
gI  = ideal gens gb  I;
ggI = ideal gens gb gI;
gI
ggI 
```

**Comment by jakobkroeker** [comment]
assign milestone 'version 1.9' instead of 'version 1.8' for all issues over ZZ?

**Comment by DanGrayson** [comment]
I think Mike hopes to fix them before 1.8.  On the other hand, I hope to release 1.8 now.

**Comment by mikestillman** [comment]
This might be a good idea.  It will take a bit more effort to fix these.

**Comment by DanGrayson** [comment]
Mike, go ahead now and retarget for 1.9 the ones you will not have time to
fix.

**Comment by mikestillman** [comment]
Fixed in commit 389dc05451a7d0b1703cdf8b9e660de76e4d791b.  These tests were incorporated into a test file test/gbZZ-2.m2
